lib.php

```diff
@@ -47,6 +47,7 @@ function accredible_add_instance($post) {
     $groupid = $localgroups->sync_group_with($course, $post->instance, $post->groupid);
 
     $localcredentials = new credentials();
+    $evidenceitems = new evidenceitems();
```

Comment:
This will throw a not found exception as the class `evidenceitems` are not included in this file.
Add - `use mod_accredible\local\evidenceitems;`